docs/understanding-tracking-design/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "🆕 Managing tracking scenarios in the console"
+date: "2020-04-17"
+sidebar_position: 95
+---
+
+Configure specific implementation rules, entities and triggers for your tracking, boosting decision making and collaboration among team members through BDP Console.
+
+## Creating a New Tracking Scenario draft
+
+To create a new Tracking Scenario, first navigate to **Data Structures** in the menu. From there, select a data structure from the list.
+
+:::info
+Tracking Scenarios can only be created from Data Strucutres that have been set as **Event**.
+:::
+
+After selecting a data structure, click on the **Tracking Scenarios** tab as shown in the image.
+
+![](images/ts-1.png)
+
+Next, click the **'Add new scenario'** button. A new modal will appear with fields for basic information that can be defined during creation, as shown.
+
+![](images/ts-2.png)
+
+Be aware of the **Show version** of the data structure against which the Tracking Scenario will be created. This means that the tracking will be compatible with this specific Data Structure version but may not be compatible against other versions or future versions if they evolve.
+
+The fields available to fill at this stage are:
+
+- A text field with the **Scenario name**. 
+- A tag field to indicate which applications this scenario is design to tracked. This will allow to create or reuse existing tags from other defined tracking scenarios.
```

Comment:
```suggestion
- A tag field to indicate which applications this scenario is designed to track. This will allow to create or reuse existing tags from other defined tracking scenarios.
```